gremlin-server/src/test/java/org/apache/tinkerpop/gremlin/server/GremlinServerHttpIntegrateTest.java

```diff
@@ -46,6 +46,7 @@
 import java.util.Base64;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.concurrent.*;
```

Comment:
Nit: In general I think wildcard imports are discouraged.

Reply:
I'll change my IDE settings so it doesn't do this anymore.